### DISCUSSION TITLE
Is there a way to display a note's children even when it has contents?

### DISCUSSION THREAD

**Discussion by veritatis** [TriliumNext/Trilium#1454] [Q&A]
The feature "This note doesn't have any content so we display its children" is brilliant ~ 
However, it seems that, I have to intentionally leave a note blank to see the thumbnails of its leaf notes. Is it possible (maybe via a label) to toggle this feature on/off manually? Then I can write a few sentences for a parent note and see its children's thumbnails in the mean time. Thanks a lot!

**Comment by zadam** [comment]
Hi, this is not possible ATM, but this has been asked for before and might be added at some point.
